contracts/AelinDeal.sol

```diff
@@ -132,6 +139,11 @@ contract AelinDeal is AelinERC20, IAelinDeal {
                 openRedemption.start = proRataRedemption.expiry;
                 openRedemption.expiry = proRataRedemption.expiry + openRedemption.period;
             }
+
+            address aelinEscrowStorageProxy = _cloneAsMinimalProxy(aelinEscrowAddress, "Could not create new escrow");
+            aelinFeeEscrow = AelinFeeEscrow(aelinEscrowStorageProxy);
+            aelinFeeEscrow.initialize(aelinTreasuryAddress, underlyingDealToken);
+
```

Comment:
should we maybe emit an event after this? for the escrow address?

Reply:
Yeah I think it makes sense

Reply:
see 4f84e871ad166bfdfc65e068ffa1b54f027c06bb